css/common.css

```diff
@@ -105,7 +105,7 @@ header .separator {
 /* --- Common Footer Styles --- */
 
 footer {
-    background-color: #e6e6e6;
+    background: none;
```

Comment:
this was to prevent weird issues when the body text was so short that the footer would clip into the dark region. this should be re-added and set to the [new body color](https://github.com/pebble-dev/pebble-dev.github.io/pull/11/files#diff-7a7ef99b6b6f3fe33c513daab1e2d4b4R42)